src/input_common/sdl/sdl_impl.cpp

```diff
@@ -594,8 +594,18 @@ class SDLButtonPoller final : public SDLPoller {
         while (state.event_queue.Pop(event)) {
             switch (event.type) {
             case SDL_JOYAXISMOTION:
-                if (std::abs(event.jaxis.value / 32767.0) < 0.5) {
+                if (axis_memory.find(event.jaxis.axis) == axis_memory.end()) {
```

Comment:
```suggestion
                if (!axis_memory.count(event.jaxis.axis)) {
```
And invert the clauses.